magic_filter/magic.py

```diff
@@ -103,23 +67,29 @@ def endswith(self, value: str) -> AttrOperation:
         """
         return AttrOperation(operation=str.endswith, value=value, magic=self)
 
-    def func(self, func: Callable[..., Any]) -> FuncOperation:
+    def func(self, value: Callable[..., Any]) -> FuncOperation:
         """
         Execute any callable on value
         """
-        return FuncOperation(value=func, magic=self)
+        return FuncOperation(value=value, magic=self)
 
-    def all(self, *call: Callable[[Any, DataType], bool]) -> AllFuncOperation:
+    def all(self, *call: Callable[[Any, Any], bool]) -> AllFuncOperation:
```

Comment:
Data argument removed. In this case signature can be changed to:
```suggestion
    def all(self, *call: Callable[[Any], bool]) -> AllFuncOperation:
```